data/issueapi.go

```diff
@@ -101,8 +99,16 @@ func FetchIssues(query string, limit int, pageInfo *PageInfo) (IssuesResponse, e
 
 	issues := make([]IssueData, 0, len(queryResult.Search.Nodes))
 	for _, node := range queryResult.Search.Nodes {
+		if node.Issue.Repository.IsArchived {
+			continue
+		}
 		issues = append(issues, node.Issue)
 	}
+
+	sort.Slice(issues, func(i, j int) bool {
```

Comment:
Can you please create an issue that removes the sorting code, and adds a default `sort:updated` like in https://github.com/dlvhdr/gh-dash/pull/223

Reply:
I'm not sure I follow the reference - this issue is `#223` and when I searched existing issues and PRs for `sort` I don't see any other issue that I could reference.

Reply:
I swear I pasted a link to the Github filters docs :D

Reply:
done: https://github.com/dlvhdr/gh-dash/issues/225

Reply:
Thanks for review!
I added sort qualifier to `makeIssuesQuery` and `makePullRequestsQuery` .